src/CMakeLists.txt

```diff
@@ -476,7 +472,6 @@ add_executable(${EXECUTABLE_GTESTS}
                test/unit/math/IndexUnitTest.cpp
                test/unit/math/MathsTest.cpp
                test/unit/math/SparseBinaryMatrixTest.cpp
-               test/unit/math/SparseMatrix01UnitTest.cpp
```

Comment:
please hold off the removals to Matrix related tests, as https://github.com/numenta/nupic.core/issues/376